modAionImpl/src/org/aion/zero/impl/sync/SyncStats.java

```diff
@@ -230,6 +230,74 @@ public void updatePeerTotalBlocks(String _nodeId, int _totalBlocks) {
         }
     }
 
+    /**
+     * Updates the total number of blocks imported from each seed peer
+     *
+     * @param _nodeId peer node display Id
+     * @param _importedBlocks total number of blocks imported
+     */
+    public void updatePeerImportedBlocks(String _nodeId, int _importedBlocks) {
+        seedsLock.lock();
```

Comment:
please update this method to do nothing when `seedEnabled` is `false` (similar to `updatePeerTotalBlocks`)